Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM php:latest
+
+ARG WORKDIR="/code"
+
+WORKDIR $WORKDIR
+
+COPY . $WORKDIR/
+
+RUN apt-get update && apt-get install -y \
```

Comment:
What do you think of install OhMyZsh?

Reply:
Yes I tried but I didn't get it :(

Reply:
Yes, I tried with the one from the uni but it gave me an error, I've already fixed it :)